frontend/tasks/content.js

```diff
@@ -98,9 +111,27 @@ function buildExperimentsData() {
     cb();
   }
 
+  function extractNewsUpdateStrings() {
+    // Extract FTL strings for news updates
+    const newsUpdateL10nFields = ['title', 'content'];
+    newsUpdates.forEach(update => {
+      newsUpdateL10nFields.forEach(fieldName => {
+        strings.push({
+          key: util.newsUpdateL10nId(update, fieldName),
+          value: update[fieldName]
+        });
+      });
+    });
+  }
+
   function endStream(cb) {
+    // One last sweep through updates to exclude dev content where not wanted
+    newsUpdates = newsUpdates
+      .filter(update => config.ENABLE_DEV_CONTENT || !update.dev);
+    extractNewsUpdateStrings();
+
     this.push(new gutil.File({
-      path: 'static/locales/en-US/experiments.ftl',
+      path: '../../locales/en-US/experiments.ftl',
```

Comment:
This is a not-great hack in lieu of a real solution for #1781 - it spits the FTL outside the build directory and into `locales/en-US/experiments.ftl` because the [`assets-locales` task](https://github.com/mozilla/testpilot/blob/c1f139a2346b08b3b839520f7523cf4f0acee903/frontend/tasks/assets.js#L4) competes with this one and clobbers the strings extracted from news updates and experiments. (Which is quite visible when strings are brand new.)